ui/shared/components/panel/variants/VariantIndividuals.jsx

```diff
@@ -377,10 +363,166 @@ const Genotype = React.memo(({ variant, individual, isCompoundHet, genesById })
           {filters}
         </small>
       )}
+    </div>
+  )
+}
+
+GenotypeQuality.propTypes = {
+  genotype: PropTypes.object,
+  variant: PropTypes.object,
+}
+
+const LegacyAdditionalSampleTypePopup = ({ genotype, variant, isHemiX, genesById }) => {
```

Comment:
We should never be maintaining a special legacy UI unless there is an overwhelming reason not to use the new UI. You should be transforming the `otherSample` into an array of genotypes with both samples and using the new, supported UI

Reply:
I added otherSample to the genotypes array, screenshot incoming.

Reply:
`otherSample` is not part of any saved variant json, it is [added](https://github.com/broadinstitute/seqr/blob/master/seqr/utils/search/elasticsearch/es_search.py#L1171) to the genotype inside of the es_utils code path. Is this something I can test?

Reply:
Hmm okay so not easy to test but we should keep support for it, but we do not need to keep the current UI for it. I think changing the code to move it into the the genotypes array is sufficient here, no screenshot needed